app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  validates :content, presence: true
+  validates :author, presence: true
```

Comment:
We don't need to validate author here as we are already saying `:belongs_to :author`. The association there already does the validation for you